lib/create-disputed-vote.js

```diff
@@ -0,0 +1,101 @@
+const fs = require('fs')
+const { utf8ToHex } = require('web3-utils')
+const { bn, getEventArgument } = require('@aragon/contract-helpers-test')
+
+module.exports = async function createDisputedVote(disputePath) {
+  const { agreement, voting, script, submitter, challenger } = await loadDisputeConfig(disputePath)
+
+  await signAgreementIfNecessary(agreement, submitter.address)
+  const actionId = await newAction(agreement, voting, script, submitter)
+  const { arbitrator, feeToken, feeAmount } = await fetchArbitratorInfo(agreement, actionId)
+  await challenge(agreement, actionId, arbitrator, feeToken, feeAmount, challenger)
+  await dispute(agreement, actionId, arbitrator, feeToken, feeAmount, submitter)
+}
+
+async function newAction(agreement, voting, script, submitter) {
+  console.log('\nCreating vote action...')
+  const receipt = await voting.newVote(script, utf8ToHex(submitter.context), { from: submitter.address })
+  const actionId = getEventArgument(receipt, 'ActionSubmitted', 'actionId', { decodeForAbi: agreement.abi })
```

Comment:
Oh interesting, I never considered this as a usecase when working with scripts. Even if you use a web3/ethers contract object, you don't get this event decoded because it comes from the Agreement?

Reply:
exactly :)